app/src/main/res/values-ru/moex_strings.xml

```diff
@@ -10,26 +10,26 @@
   <string name="MoexUpdatesText">Обновления</string>
   <string name="MoexSourceText">Исходный код</string>
   <string name="MoexCategories">Категории</string>
-  <string name="MoexHideButtons">Панель сообщения</string>
-  <string name="DisableCommandsButton">Скрыть кнопку Комманд</string>
-  <string name="DisableCameraButton">Скрыть кнопку Камеры</string>
-  <string name="HideCameraButtonInfo">Эта функция так же добавляет плавающую кнопку камеры в меню отправки медиа</string>
-  <string name="DisableRecordButton">Скрыть кнопку Медиа</string>
-  <string name="moexNews">Новости о Moegram-X</string>
+  <string name="MoexHideButtons">Поле ввода сообщений</string>
+  <string name="DisableCommandsButton">Скрыть кнопку команд</string>
+  <string name="DisableCameraButton">Скрыть кнопку камеры</string>
+  <string name="HideCameraButtonInfo">Опция выше также добавляет плавающую кнопку камеры в меню отправки медиа</string>
+  <string name="DisableRecordButton">Скрыть кнопку аудио-/видеосообщений</string>
```

Comment:
Hi! Thanks for the contribution, but I think that this string should be renamed as it takes too many space

![image](https://user-images.githubusercontent.com/61517619/230783906-2a0fc871-07c8-4bd3-a6fd-a9afe0cb74c8.png)